.github/workflows/ci.yaml

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: CI
+# Trigger the workflow on push
+on:
+  push:
+    branches:
+      - master
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE: eclipse/che-dashboard
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout che-dashboard source code
+    - uses: actions/setup-node@v1
+      name: Configuring nodejs 8.x version
+      with:
+        node-version: '8.x'
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - uses: actions/cache@v1
+      id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    - name: Install
+      run: yarn install --ignore-optional
+    - name: Build
+      run: yarn build -t eclipse-che-dashboard
+    - name: Test
+      run: yarn test
+    - name: "Docker build"
+      run: |
+        docker image prune -a -f
+        docker build . -f Dockerfile.apache -t quay.io/${DOCKER_IMAGE}:latest
```

Comment:
hi, maybe should use another tag than latest ?
like `short-sha1 of git` or `next` ?